docker/wc-admin-php-test-suite/entrypoint.sh

```diff
@@ -47,9 +60,26 @@ else
 	fi
 fi
 
+
 # Run the install script if the WordPress directory is not found.
 if [ ! -d /tmp/wordpress-tests-lib ]; then
 	install
 fi
 
-exec phpunit "$@"
+if [ "$(php -r "echo version_compare(PHP_VERSION,'8','>=');")" ]; then
+	composer install
```

Comment:
Do we need to run `composer install` here? I think ideally we would run it outside the container so that the tests can start sooner.

It also produced an error for me (see below). I got around it by running `composer install` from the host.

```
A script named bin would override a Composer command and has been skipped
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - squizlabs/php_codesniffer is locked to version 3.5.8 and an update of this package was not requested.
    - squizlabs/php_codesniffer 3.5.8 requires ext-simplexml * -> it is missing from your system. Install or enable PHP's simplexml extension.
  Problem 2
    - squizlabs/php_codesniffer 3.5.8 requires ext-simplexml * -> it is missing from your system. Install or enable PHP's simplexml extension.
    - wp-coding-standards/wpcs 2.3.0 requires squizlabs/php_codesniffer ^3.3.1 -> satisfiable by squizlabs/php_codesniffer[3.5.8].
    - wp-coding-standards/wpcs is locked to version 2.3.0 and an update of this package was not requested.
```

Reply:
Thanks, we indeed don't need to. I originally had it when the bin package wasn't part of the `composer.json`, but as long as you have run `composer install` locally it's all good.